src/models/multiChart.js

```diff
@@ -419,6 +419,9 @@ nv.models.multiChart = function() {
 
                     interactiveLayer.tooltip
                     .chartContainer(chart.container.parentNode)
+                    .headerFormatter(function(d, i) {
+                        return xAxis.tickFormat()(d, i);
+                    })
```

Comment:
The problem with this (and the valueFormatter below) is that if someone does:

    chart.interactiveGuideline.tooltip.headerFormatter(function(){ return 'a'; });    

I think you should be able to do ._headerFormatter(...) and ._valueFormatter(...), if you start an option with an underscore then it tries to only use the passed value if it wasn't overwritten already